### ISSUE TITLE
q re Skyrim AE support

### ISSUE THREAD

**Issue by SteveTownsend** [Ryan-rsm-McKenzie/CommonLibSSE#115]
Hey, sorry for the ping - is master branch compatible with both AE and SE binaries now? I see some relevant updates, but don't want to move my mod to latest unless it's going to work on both.

**Comment by SteveTownsend** [comment]
Ugh, this report seems way more entitled than I intended. I just wanted to ask about status, since users are complaining and I saw some work had been done that looked related - it's definitely not a demand for an upgrade path, soon or ever, unless it's on your radar already.

**Comment by Ryan-rsm-McKenzie** [comment]
It only supports AE now

**Comment by SteveTownsend** [comment]
Thanks. So for back-level compat I should freeze my current local, by the
sound of things. I'll need to build two separate versions going
forwards, at least until all the mods I use on SE support AE.

Sorry for the lame "noob q in a Github issue".

**Comment by SteveTownsend** [comment]
Just to update here. Thanks so much to you and all who worked on getting this ready so quickly. Seems like my mod is now working OK on AE. FYI I made a fork of your master that incorporates some local changes I keep around because I could not be bothered to sync to all your latest: I never yet got around to using CMake for home desktop dev, and I could not get the _SKSE::WinAPI_ entry points to resolve properly so I just prepended _CLSSE_ to the affected function names. The main thing I did was make it buildable for both SE and AE using a compiler macro _SKYRIM_AE_ , since I don't expect to be on AE myself for months, if ever.
